maas/resource_maas_vm_host.go

```diff
@@ -225,13 +227,9 @@ func resourceVMHostUpdate(ctx context.Context, d *schema.ResourceData, meta inte
 	if err != nil {
 		return diag.FromErr(err)
 	}
-	vmHost, err := client.VMHost.Get(id)
-	if err != nil {
-		return diag.FromErr(err)
-	}
 
 	// Update VM host options
-	_, err = client.VMHost.Update(vmHost.ID, getVMHostParams(d))
+	_, err = client.VMHost.Update(id, getVMHostParams(d))
```

Comment:
My understanding that this change is just an optimisation to remove extra API call under `client.VMHost.Get`?

Reply:
This is true, yes